lms/migrations/versions/b512a5cf64ed_public_id_migration.py

```diff
@@ -0,0 +1,30 @@
+"""Public ID migration.
+
+Revision ID: b512a5cf64ed
+Revises: 329313b38de1
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+revision = "b512a5cf64ed"
+down_revision = "329313b38de1"
+
+
+def upgrade() -> None:
+    conn = op.get_bind()
+    conn.execute(
+        sa.text(
+            """UPDATE "organization" set public_id = 'us.lms.org.' || "public_id";"""
```

Comment:
TODO: this needs  to read region_code or it will change all envs to the same value.